tests/examples/python/python_test.go

```diff
@@ -288,7 +288,8 @@ func TestHelm(t *testing.T) {
 		t.FailNow()
 	}
 	options := baseOptions.With(integration.ProgramTestOptions{
-		Dir: filepath.Join(cwd, "helm"),
+		Dir:                  filepath.Join(cwd, "helm"),
+		ExpectRefreshChanges: true,
```

Comment:
What triggered the expectancy of `refreshChanges` here?

Reply:
This chart includes a `PodDisruptionBudget` that has a status update once the Deployment rolls out. This causes changes on refresh. It _would_ be nice to have an option to ignore only status changes, but there's currently no way to do that with the test runner.